pkg/saved_objects/saved_objects.go

```diff
@@ -66,7 +66,7 @@ func (p *SavedObjectsProvider) GetObject(ctx context.Context, obj *SavedObjectOS
 		return nil, nil
 	}
 	if res.StatusCode != http.StatusOK {
-		return nil, diag.FromErr(fmt.Errorf("request failed, statuscode %d", err))
```

Comment:
isn't the current version on the main branch already sufficient?
https://github.com/moia-oss/terraform-provider-opensearch-dashboards/blob/d33d97fc09e1a1c527ae7e47cf2f1fa6fe2092cf/pkg/saved_objects/saved_objects.go#L69